### ISSUE TITLE
Hosts/Guests pages - make layout identical

### ISSUE THREAD

**Issue by gerbrent** [JupiterBroadcasting/jupiterbroadcasting.com#145]
* [ ] Apply changes made previously to Hosts page design/layout apply to Guest pages

For example, see differences:
* [hosts/alex](https://jupiterbroadcasting.net/hosts/alex/)
* [guests/alex](https://jupiterbroadcasting.net/guests/alex/)

### hosts/alex
![image](https://user-images.githubusercontent.com/7018928/180285054-c4dfe85f-467d-4195-a57c-476b31769985.png)

### guests/alex
![image](https://user-images.githubusercontent.com/7018928/180285142-d57dd43e-f8d0-4dc5-b205-c34e51ca5723.png)

**Comment by gerbrent** [comment]
@elreydetoda , @[Dedpy](https://github.com/Dedpy) ... interested in working on this one?

**Comment by elreydetoda** [comment]
Sure @gerbrent , I can take this one.

**Comment by Lord0fBytes** [comment]
@elreydetoda If you are still working on this one, I updated the hosts page in PR #228 to adjust the layout and aspect ratio of the cards. Feel free to copy the layout for the guests or let me know if you want some help.

**Comment by elreydetoda** [comment]
That sounds good, thanks @Lord0fBytes , I'll be sure to checkout the hosts page and match them. 😁

**Comment by gerbrent** [comment]
In the Hugo spirit of "everything only exists once" - is there a different way then to define the Host and Guest pages to always be identical in functionality?

**Comment by elreydetoda** [comment]
Ideally...yes, but I'll need to check in the code (on mobile now) to see what's currently prohibiting that from happening now.

The only thing that I could think of is if the keys in the json are different. Even so, we should be able to handle that just not as easily if they were the same.

I'll work towards that with this issue though.

**Comment by elreydetoda** [comment]
Alright, for now I think I've replicated what @Lord0fBytes did for the host page to the guest page. I'll wait till he replies to #241 to actually make the PR ready to merge, so the pages are consistent (with or without the `-`). Here is a quick screenshot of pages.

Host page
![image](https://user-images.githubusercontent.com/10230166/183336458-987026d8-ef12-488e-a248-14edb0faf2fb.png)

Guest page
![image](https://user-images.githubusercontent.com/10230166/183336472-c97b6a67-e54f-4383-ac33-7ba8863e204b.png)


> In the Hugo spirit of "everything only exists once" - is there a different way then to define the Host and Guest pages to always be identical in functionality?

tl;dr: so ya it is more complicated for now. Once we do #110 though it'll be easier.

Longer answer:
Currently we're using taxonomies to render out the current host & guest info, at least that's my understanding of it, and once we finish doing #110 we'll be able to just do `single.html` pages for those individual page and a `list.html` to show all the people.
To keep them in sync we can just symlink one file to another, and so if one changes the other one will automatically get it (in theory :upside_down_face: ).

So, we'll cross that bridge when we get there, but till then we can't keep them unified/synced.

**Comment by Lord0fBytes** [comment]
That looks great!

**Comment by elreydetoda** [comment]
Well I had a really good design to base it off of 😁, cool I'll get the other PR ready for merge as well then.
